src/components/Home.js

```diff
@@ -0,0 +1,134 @@
+
+import React from 'react'
+import styled from 'styled-components';
+import banners from '../data/featured-banners.json'
+import productCategories from '../data/product-categories.json'
+import FeaturedProducts from './FeaturedProducts';
+
+
+const DivCategory = styled.div`
+        border: 1px solid;
+        min-width: 30vh;
+        background-color: #5472d3;
+    `;
+
+const ImageInSlider = styled.img`
+        max-height: 100vh;
+        width: 100vh;
+        `;
+
+
+function Categories(){
+    const categories = productCategories.results.map((obj) => {
+        return obj.data.name
+    });
+    const [categoryItems,setCategoryitems] = React.useState(categories.slice(0,3));
```

Comment:
and here only use something like this:
```suggestion
    const [categoryItems,setCategoryitems] = useState(categories.slice(0,3));
```